docs/source/getting-started/installing.adoc

```diff
@@ -42,8 +42,7 @@ Windows::
 +
 [NOTE]
 ====
-To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] using the Hyper-V Manager.
-For specific configuration steps see the xref:../getting-started/setting-up-driver-plugin.adoc#[Setting Up the Driver Plug-in] section.
+To use {project} with Hyper-V ensure that, after you link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/enable-hyper-v[install Hyper-V], you also link:https://docs.microsoft.com/en-us/virtualization/hyper-v-on-windows/quick-start/connect-to-network[add a Virtual Switch] using the Hyper-V Manager and set the environment variable `HYPERV_VIRTUAL_SWITCH` to this virtual switch. For specific configuration steps see the xref:../getting-started/setting-up-driver-plugin.adoc#[Setting Up the Driver Plug-in] section.
```

Comment:
So we need to set the 'HYPERV_VIRTUAL_SWITCH' variable? That sucks? What happened with the driver trying to pick one? Is this disabled?

Reply:
@budhrg @gbraad ^^ ?

Reply:
> So we need to set the 'HYPERV_VIRTUAL_SWITCH' variable? That sucks? What happened with the driver trying to pick one? Is this disabled?

It seemed so. I guess @gbraad added this behavior in order to avoid any sort of inconsistency.

Reply:
> It seemed so

It seems so or it is? Do we really want to force the use of HYPERV_VIRTUAL_SWITCH?

Reply:
@anjannath, on a different note, one sentence a line ;-) You need to start a new line before "For specific ..."

Reply:
This has been discussed and agreed on before...

"The automatic selection can cause issues as the FIRST switch is always chosen when not explictly set. This means that VERY often the DockerNAT is the first switch to be used". So, yes, this is now as expected.

I would suggest implementing a flag `--hyperv-virtual-switch` going forward.

Reply:
> I would suggest implementing a flag --hyperv-virtual-switch going forward.

Environment variable approach was preferred over it here https://github.com/minishift/minishift/issues/739 as it also allows to have other options as well.

Reply:
Which makes no sense in light of the profile functionality. When you create an instance, you might want to assign it to another VSwitch, and with a specific `--network-ipaddress` and `--network-gateway`.

Reply:
Fix #590 is in my opinion preferred... Solution #739 was chosen on the basis that #305 would take 'some' time => actually, the switch option should have been added here:

https://github.com/minishift/minishift/pull/773/files#diff-221fdad1b39f75fb2d0214c25944dcb3R421

as part of #773.